<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="NameLabel">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="task1">
          <property name="text">
           <string>Задача 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="task2">
          <property name="text">
           <string>Задача 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="task3">
          <property name="text">
           <string>Задача 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="task4">
          <property name="text">
           <string>Задача 4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="task5">
          <property name="text">
           <string>Задача 5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="stat_label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Статистика&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#aa00ff;&quot;&gt;Задача 1     3&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#aa00ff;&quot;&gt;Задача 2    -1&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#aa00ff;&quot;&gt;Задача 3     5&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#aa00ff;&quot;&gt;Задача 4     0&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Выйти</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>О программе</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_1">
   <property name="text">
    <string>Задача 1</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Задача 2</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Задача 3</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Задача 4</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Задача 5</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
